amical/get_infos_obs.py

```diff
@@ -231,3 +231,79 @@ def get_pixel_size(ins):
     except KeyError:
         p = np.NaN
     return p
+
+
+def get_ifu_table(
+    i_wl, filtname="YH", instrument="SPHERE-IFS", verbose=False, display=False
+):
+    """Get spectral information for the given instrumental IFU setup.
+    `i_wl` can be an integer, a list of 2 integers (to get a range between those
+    two) or a list of integers (>= 3) used to display the
+    requested spectral channels."""
+    from matplotlib import pyplot as plt
+
+    wl = get_wavelength(instrument, filtname) * 1e6
+
+    if np.isnan(wl.any()):
+        return None
+
+    if verbose:
+        print(f"\nInstrument: {instrument}, spectral range: {filtname}")
+        print("-----------------------------")
+        print(
+            "spectral coverage: %2.2f - %2.2f µm (step = %2.2f)"
+            % (wl[0], wl[-1], np.diff(wl)[0])
+        )
+
+    one_wl = True
+    multiple_wl = False
+    if isinstance(i_wl, list) & (len(i_wl) == 2):
```

Comment:
`and` isn't syntactic sugar for `&`, they are fundamentally different operators.
```suggestion
    if isinstance(i_wl, list) and len(i_wl) == 2:
```